net.sonobus.SonoBus.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: net.sonobus.SonoBus
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: sonobus
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/Downloads
+  - --filesystem=~/Music
+# Jack
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+modules:
+  - shared-modules/linux-audio/jack2.json
```

Comment:
you must add a `cleanup` section to the whole package to remove `/lib/libjack*` and `/lib/jack`

Otherwise JACK won't work.